components/tidb_query_vec_expr/src/impl_compare.rs

```diff
@@ -264,10 +264,91 @@ pub fn greatest_int(args: &[Option<&Int>]) -> Result<Option<Int>> {
     do_get_extremum(args, max)
 }
 
+#[rpn_fn(nullable, varg, min_args = 2)]
+#[inline]
+pub fn least_int(args: &[Option<&Int>]) -> Result<Option<Int>> {
+    do_get_extremum(args, min)
+}
+
+#[rpn_fn(nullable, varg, min_args = 2)]
+#[inline]
+pub fn interval_int(args: &[Option<&Int>]) -> Result<Option<Int>> {
```

Comment:
Please add some comment to the exported function to make its meaning more clear since we could not find such function in MySQL's docs.

Reply:
> Please add some comment to the exported function to make its meaning more clear since we could not find such function in MySQL's docs.

This function is a direct mapping of https://github.com/pingcap/tipb/blob/f31a15d98fceea315accb67a53b2416d43364e33/proto/expression.proto#L177 , if there should be somewhere to leave comments, maybe in repository tipb or tidb is better?

Reply:
But this function is maintaining in `TiKV` repo, we should add some docs here to make it easier to maintain. BTW, why not use `binary_search` function that provided by the standard library.